app/views/exercises/show.html.haml

```diff
@@ -1,3 +1,4 @@
+- view_name = "#{@exercise.exercise_category.name.downcase.gsub(' ', '_')}_#{@exercise.id}"
```

Comment:
Variables should be kept out of the views. You can put this in a helper, decorator or somewhere else, where we can extract it from from the exercise it self. Maybe `exercise.name_of_partial` or something along those lines.